modules/tile-converter/test/i3s-converter/helpers/gltf-attributes.spec.js

```diff
@@ -209,6 +191,24 @@ test('gltf-attributes - Should generate attributes object from tileContent with
         }
       }
     ],
+    images: [
+      {
+        width: undefined,
```

Comment:
It looks weird to explicitly set these to undefined instead of omitting them. That would normally never happen. Maybe comment on what we are testing?